src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -26,6 +27,8 @@ def __init__(self, **kwargs):
             import logging
             kwargs['logger'] = logging
         self.logger = kwargs['logger']
+        self.rucio = Rucio(kwargs.get("rucioAccount", "wmcore_transferor"),
```

Comment:
I think so. But we have so many services being initialized here without being enclosed, which makes me thing it's not a problem (especially because we are talking about code that's here for many years)

Reply:
Ok